lblod/frontend-lokaal-mandatenbeheer
# Lmb 892 | different home page for admin

## Description

When logged in as an administrator you see the same modules as you are logged in as a normal bestuurseenheid. This would be okey if you could click on those modules without getting an error.

Show a different home page for a user that is logged in as an administrator.

## How to test

- Log in as a bestuurseenheid => everything works as before
- Log in as an administrator
  -  you should see a different home page
  - you can directly simulate another bestuurseenheid 
  - you can set a system message for all bestuurseenheden

## Attachments

**Logged in as an admin**
![image](https://github.com/user-attachments/assets/dbe1bc32-e6a3-49c6-ba54-eb9744bd6e76)

**As an admin that simulates another bestuurseenheid**
![image](https://github.com/user-attachments/assets/81aceed7-dd81-44ff-9874-39660a2f4eef)

**As normal bestuurseenheid**
![image](https://github.com/user-attachments/assets/41334c48-95e3-46d3-ab23-5cf157d26ba2)
